CHANGELOG.md

```diff
@@ -57,6 +57,7 @@
 * [7414](https://github.com/grafana/loki/pull/7414) **thepalbi**: Add basic tracing support
 
 ##### Fixes
+* [7394](https://github.com/grafana/loki/pull/7394) **liguozhong**: fix(cloudflare): add Promtail cloudflare close. Fixes Promtail Cloudflare logpull stops working after a period of time in the issue https://github.com/grafana/loki/issues/6150
```

Comment:
Suggestion to make this a bit clearer. Also I think we don't need the `fix` prefix, because it's listed under the section `Fixes`.

```suggestion
* [7394](https://github.com/grafana/loki/pull/7394) **liguozhong**: Fix issue with the Cloudflare target that caused it to stop working after it received an error in the logpull request as explained in issue https://github.com/grafana/loki/issues/6150
```